boards/prototype/base-workloads/br-chip-base.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name" : "br-chip-base",
+  "distro" : {
+      "name" : "br",
+      "opts" : {
+          "configs" : ["buildroot-config"]
+      }
+  },
+  "overlay" : "overlay",
+  "linux" : {
+      "source" : "../../linux",
+      "config" : "linux-config"
+  },
+  "firmware" : {
+      "use-bbl" : false,
+      "bbl-src" : "../../firmware/riscv-pk",
```

Comment:
Does BBL work at all or do you just prefer not to use it by default?

Reply:
BBL works. When I originally set this up I was toggling between the two to find out why things weren't booting. Are you advocating to clean up things a bit?

Reply:
FYI, the reason FireMarshal didn't originally work on the FPGA was SDCard space issue (the chip bootrom would load in 20MiB exactly and the FireMarshal payload was slightly larger than 20MiB...)